app/src/features/FhirAttributePanel/AttributeInputGroup.tsx

```diff
@@ -87,8 +89,22 @@ const AttributeInputGroup = ({
     popupId: "popup",
   });
   const [deleteInputGroups] = useApiInputGroupsDestroyMutation();
+  const [updateInputGroups] = useApiInputGroupsUpdateMutation();
   const [createInput] = useApiInputsCreateMutation();
-  const { data: inputs } = useApiInputsListQuery({ inputGroup: inputGroup.id });
+  const { data: inputs } = useApiInputsListQuery(
+    { inputGroup: inputGroup.id },
+    {
+      selectFromResult: ({ data }) => ({
+        data: data
+          ? [...data].sort(
+              (inputA, inputB) =>
+                Date.parse(inputA.created_at) - Date.parse(inputB.created_at)
+            )
+          : undefined,
```

Comment:
can't you use the `??` syntax for this ? maybe it returns null instead of undefined ?

Reply:
`data` can be undefined and is frozen, which means I cannot mutate it with the `sort` method and I need to clone it with `...`, which doesn't work on variables that can be undefined. Am I doing this wrong ? 🧐

So this wouldn't work : 

```typescript
data?.sort(
   (inputA, inputB) =>
      Date.parse(inputA.created_at) - Date.parse(inputB.created_at)
   )
```

Reply:
In any case I think i'll have to remove this code. We may change the API list ordering to sort those by `created_at` instead of `updated_at`